protomap/proto.go

```diff
@@ -430,6 +449,20 @@ func makeWrapper(msg protoreflect.Message, fd protoreflect.FieldDescriptor, val
 			return nil, false, fmt.Errorf("got non-string value for string field, field: %s, value: %v", fd.FullName(), val)
 		}
 		return (&wpb.StringValue{Value: nsv}).ProtoReflect(), true, nil
+	case *wpb.UintValue:
+		var nsv uint64
+		switch {
+		case wasTypedVal:
+			nsv = val.(uint64)
+		default:
+			iv, ok := val.(int)
```

Comment:
typo: uint? the test should be fixed as well.

Reply:
Yes, thanks!